import.go

```diff
@@ -17,55 +19,78 @@ import (
 // @Produce  json
 // @Param import body main.Import _ "import object"
 // @Success 201 "Successfully imported"
-// @Failure 400 {object} string "Bad request"
-// @Failure 401 {object} string "Unauthorized or missing jwt token"
+// @Failure 400 {object} problem.Response "Bad request"
+// @Failure 401 {object} problem.Response "Unauthorized or missing jwt token"
+// @Failure 502 {object} problem.Response "Bad gateway"
 // @Router /gitlab [post]
 func runGitLabHandler(c *gin.Context) {
 	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
 
 	i := Import{}
 	err := c.BindJSON(&i)
-	if handleError(c, err) {
+	if err != nil {
+		ginutil.WriteInvalidBindError(c, err,
+			"One or more parameters failed to parse when reading the request body for GitHub projects import/refresh")
```

Comment:
Need to use `ShouldBindJSON` when using the `ginutil.Write...` functions

https://iver-wharf.github.io/#/development/gin-shouldbind-vs-bind